scriptmodules/ports/wolf4sdl.sh

```diff
@@ -31,14 +31,8 @@ function _get_opts_wolf4sdl() {
 }
 
 function add_games_wolf4sdl() {
+    eval "declare -A games="${2#*=}
```

Comment:
there is no need to do this - you can access games from this function if games is set in the parent with local scope.

Reply:
I originally moved the `declare -A wolf3d_games` for the array up to the top of the `splitwolf.sh` and tried to use it as a global variable, but `add_games_wolf4sdl` inside of `wolf4sdl.sh` could not access it, so instead I passed it in and used this hack to define it because you cannot pass an associative array in bash.

If there is a better way, I'll gladly do it, but I am not aware.